README.rst

```diff
@@ -25,6 +25,44 @@ The rendered version of the gallery is hosted on `readthedocs.org`_.
 Contributing
 ============
 
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `pyfar_gallery` for local development using the command-line interface. Note that several alternative user interfaces exist, e.g., the Git GUI, `GitHub Desktop <https://desktop.github.com/>`_, extensions in `Visual Studio Code <https://code.visualstudio.com/>`_ ...
+
+1. `Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo/>`_ the `gallery` repo on GitHub.
+2. Clone your fork locally and cd into the gallery directory::
+
+    $ git clone https://github.com/YOUR_USERNAME/gallery.git
+    $ cd gallery
+
+3. Install your local copy into a virtualenv. Assuming you have Anaconda or Miniconda installed, this is how you set up your fork for local development::
+
+    $ conda create --name gallery python
+    $ conda activate gallery
+    $ pip install -r requirements.txt
+
+4. Setup pre-commit hooks, this will clean all notebooks in ``docs/gallery/interactive`` before the commit::
+
+    $ pre-commit install
+
+5. Create a branch for local development. Indicate the intention of your branch in its respective name (i.e. `feature/branch-name` or `bugfix/branch-name`)::
+
+    $ git checkout -b name-of-your-bugfix-or-feature
+
+   Now you can make your changes locally.
+
+6. Commit your changes and push your branch to GitHub::
```

Comment:
Maybe add information about the pre-commit workflow:

'Note that the commit will fail if you are trying to commit notebooks including output. Trying to push will automatically remove the output in these notebooks after which they must be saved and the push can be done.'